tests/test_pipelines.py

```diff
@@ -273,7 +273,9 @@ def test_stable_diffusion_ddim(self):
         image_slice = image[0, -3:, -3:, -1]
 
         assert image.shape == (1, 128, 128, 3)
-        expected_slice = np.array([0.5112, 0.4692, 0.4715, 0.5206, 0.4894, 0.5114, 0.5096, 0.4932, 0.4755])
+        expected_slice = np.array(
+            [0.51117265, 0.4692378, 0.471545, 0.52064794, 0.48938107, 0.51137197, 0.5095732, 0.49317074, 0.47548753]
```

Comment:
Why change all the values here? Do the tests fail for you locally? (Do you use a GPU or CPU?)